src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -328,15 +306,14 @@ else if (!authCheckService.isAtLeastTeachingAssistantForExercise(fileUploadExerc
      * with status 400 (Bad Request) if the fileUploadExercise is not valid, or with status 409 (Conflict)
      * if given exerciseId is not same as in the object of the request body, or with status 500 (Internal
      * Server Error) if the fileUploadExercise couldn't be updated
-     * @throws URISyntaxException if the Location URI syntax is incorrect
      */
-    @PutMapping(Endpoints.REEVALUATE_EXERCISE)
+    @PostMapping(Endpoints.REEVALUATE_EXERCISE)
```

Comment:
did we decided to use Post for re-evaluate, because we discussed this before in [here](https://github.com/ls1intum/Artemis/pull/3521#pullrequestreview-678681984) also in [here ](https://github.com/ls1intum/Artemis/pull/3521#pullrequestreview-678749904)..  After re-evalution, it invokes updateXexercise endpoint? Will you also change this for other exercise resources?

Reply:
You are right, we agreed on PUT in this case, I'm reverting the changes.